examples/all-clusters-app/all-clusters-common/all-clusters-app.zap

```diff
@@ -5633,6 +5593,196 @@
+          "code": 43690,
+          "mfgCode": null,
+          "define": "NETWORK_PROVISIONING_CLUSTER",
+          "side": "client",
+          "enabled": 1,
+          "commands": [
+            {
+              "name": "ScanNetworks",
+              "code": 0,
+              "mfgCode": null,
+              "source": "client",
+              "incoming": 1,
+              "outgoing": 1
+            },
+            {
+              "name": "AddWiFiNetwork",
+              "code": 2,
+              "mfgCode": null,
+              "source": "client",
+              "incoming": 1,
+              "outgoing": 1
+            },
+            {
+              "name": "UpdateWiFiNetwork",
+              "code": 4,
+              "mfgCode": null,
+              "source": "client",
+              "incoming": 1,
+              "outgoing": 1
+            },
+            {
+              "name": "AddThreadNetwork",
+              "code": 6,
+              "mfgCode": null,
+              "source": "client",
+              "incoming": 1,
+              "outgoing": 1
+            },
+            {
+              "name": "UpdateThreadNetwork",
+              "code": 8,
+              "mfgCode": null,
+              "source": "client",
+              "incoming": 1,
+              "outgoing": 1
+            },
+            {
+              "name": "RemoveNetwork",
+              "code": 10,
+              "mfgCode": null,
+              "source": "client",
+              "incoming": 1,
+              "outgoing": 1
+            },
+            {
+              "name": "EnableNetwork",
+              "code": 12,
+              "mfgCode": null,
+              "source": "client",
+              "incoming": 1,
+              "outgoing": 1
+            },
+            {
+              "name": "DisableNetwork",
+              "code": 14,
+              "mfgCode": null,
+              "source": "client",
+              "incoming": 1,
+              "outgoing": 1
+            },
+            {
+              "name": "GetLastNetworkProvisioningResult",
+              "code": 16,
+              "mfgCode": null,
+              "source": "client",
+              "incoming": 1,
+              "outgoing": 1
+            }
+          ],
+          "attributes": [
+            {
+              "name": "cluster revision",
+              "code": 65533,
+              "mfgCode": null,
+              "side": "client",
+              "included": 1,
+              "storageOption": "RAM",
+              "singleton": 0,
+              "bounded": 0,
+              "defaultValue": "0x0001",
+              "reportable": 0,
+              "minInterval": 0,
+              "maxInterval": 65344,
+              "reportableChange": 0
+            }
+          ]
+        },
+        {
+          "name": "Network Provisioning",
+          "code": 43690,
+          "mfgCode": null,
+          "define": "NETWORK_PROVISIONING_CLUSTER",
+          "side": "server",
+          "enabled": 1,
```

Comment:
I was looking for the code of the server that can handle those commands and I realised the `gen/` folder of the `all-clusters-app` has not been updated despite the fact that you are enabling some clusters on this file.

Looking at your initial comment for the PR I found this sentence: "Support on end device side will be a seperate PR later.".

Enabling some clusters in this file without providing the associated `gen/` changes will basically break the build for others. I think that's because you have not provided the associated `src/app/clusters` code and so the `gen/` folder will not link without it. Maybe you can just provide a `src/app/clusters/network-provisioning` skeleton so it will make the linker happy and you can provide the implementation later ?

Reply:
Done, network provisioning cluster will be added back to all cluster apps after basic network commissioning handlers are finished.